fedimint-client-legacy/src/mint/mod.rs

```diff
@@ -383,13 +384,87 @@ impl MintClient {
         NoteIssuanceRequest::new(ctx, secret)
     }
 
+    pub async fn summary(&self) -> TieredSummary {
+        self.start_dbtx()
+            .await
+            .find_by_prefix(&NoteKeyPrefix)
+            .await
+            .fold(
+                TieredSummary::default(),
+                |mut acc, (key, _note)| async move {
+                    acc.inc(key.amount, 1);
+                    acc
+                },
+            )
+            .await
```

Comment:
Forgot to send, seems you addressed in the meantime.

Reply:
> Forgot to send, seems you addressed in the meantime.

Yeah, it's isn't implemented because it would require more refactorings, but it's certainly a possibility to keep in mind.